### PR TITLE
Implementation of admin Home Screen (no logic)

### PR DESCRIPTION
**Pull request by selmagudmundsen**
### PR Type 🍏
What kind of change does this PR introduce?
```
[ ] Bugfix
[x] Feature
[ ] API
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Documentation content changes
[ ] Tests
[ ] Other
```
[Kanban Link](https://www.notion.so/bekks/Implementere-MVP-Hjem-design-56ebb402606248e4aad218123a218183?pvs=4)

### Context 🤷‍♀️
To finish the MVP we need to create an admin panel where the admin user can set up a new game for the players of the game. Here its necessary to have an admin panel, which this is the home page or landing page for (after logging in) where you can choose whether to set up a new game or look at the leaderboard.

### What's new? 👶
A new page in the application, which can be accessed through /admin. The page has an admin navigation bar component (located in components/NavBars) and two navigation cards. More styling and scalability will be added later, including logic.

### Screenshots 🖼️
Screenshot 2024-07-02 at 12 27 27

### CODE REVIEW

File: src/components/NavBar/AdminNavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { TopNavigationItem } from '@entur/menu'
+import { Link } from 'react-router-dom'
+import EnturPartnerIconDark from '../../assets/icons/EnturPartnerDark.svg'
+
+export const AdminNavBar: React.FC = () => {
+    return (
+        <nav className="flex flex-row items-center gap-14 p-8">
+            {/* <Link to="/">
```
Comment (Magnus-Farstad) [inline src/components/NavBar/AdminNavBar.tsx:8]: Skulle vi bare fjernet kommentert ut kode?

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
 
-    useEffect(() => {
-        setBackgroundColor('bg-blue-main')
+    // useEffect(() => {
+    //     setBackgroundColor('bg-blue-main')
 
-        return () => setBackgroundColor('bg-blue-90')
-    }, [setBackgroundColor])
+    //     return () => setBackgroundColor('bg-blue-90')
+    // }, [setBackgroundColor])
```
Comment (Magnus-Farstad) [inline src/pages/index.tsx:13]: Kan dette fjernes?

File: src/components/NavBar/AdminNavBar.tsx (hunk 1 above)
Comment (selmagudmundsen) [reply src/components/NavBar/AdminNavBar.tsx:8]: Ja, fiksa nu

File: src/pages/index.tsx (hunk 2 above)
Comment (selmagudmundsen) [reply src/pages/index.tsx]: Yes, la det inn igjen. Er styling som vi vurderte å fjerne men har valgt å beholde per nå.

**Review by Magnus-Farstad — APPROVED** [review]

**Pull request merged**